bitnami/common/templates/_capabilities.tpl

```diff
@@ -0,0 +1,11 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Return the appropriate apiVersion for deployment.
+*/}}
+{{- define "common.capabilities.deployment.apiVersion" -}}
+{{- if semverCompare "<1.14-0" .Capabilities.KubeVersion.GitVersion -}}
+{{- print "extensions/v1beta1" -}}
+{{- else -}}
+{{- print "apps/v1" -}}
+{{- end -}}
+{{- end -}}
```

Comment:
This is great but we should extend this to other K8s objects: `PodSecurityPolicy`, `NetworkPolicy`

Reply:
Right totally agree, I think we could add it in an "on demand" approach. I mean, when we find a use case for `PodSecurityPolicy` or `NetworkPolicy`

Reply:
We already have cases for this (also jobs):
- https://github.com/bitnami/charts/blob/master/bitnami/elasticsearch/templates/cronjob.yaml
- https://github.com/bitnami/charts/blob/master/bitnami/redis/templates/psp.yaml
- https://github.com/bitnami/charts/blob/master/bitnami/postgresql/templates/networkpolicy.yaml